.github/workflows/publish_docker.yml

```diff
@@ -1,21 +1,89 @@
 name: Publish Build Docker Images
 on:
   workflow_dispatch:
+    inputs:
+      build-base:
+        description: "Build 'build-base' image"
+        type: boolean
+        required: true
+        default: false
+      build-geosupport:
+        description: "Build 'build-geosupport' image"
+        type: boolean
+        required: true
+        default: false
+      dev:
+        description: "Build 'dev' image"
+        type: boolean
+        required: true
+        default: false
+      docker-geosupport:
+        description: "Build 'docker-geosupport' image"
+        type: boolean
+        required: true
+        default: false
   
 jobs:
-  build:
+  build-base:
```

Comment:
These are done explicitly instead of with matrix because I simply could not figure out how to conditionally run matrix jobs in a simple manner - it's not supported by default